### DISCUSSION TITLE
Properly changing window title text color

### DISCUSSION THREAD

**Discussion by 0xsch1zo** [ArthurSonzogni/FTXUI#939] [Q&A]
This might be a very simple question and I might be just missing something obvious, I tried and couldn't resolve my issue. When I try to make a window that has a title that's supposed to be colored and apply the color to the title the whole upper border of the window changes color:
![20241020_17h22m27s_grim](https://github.com/user-attachments/assets/92377408-78a9-4531-a6b6-8356d272c895)
Here's the code:
```cpp
#include <ftxui/dom/elements.hpp>
#include <ftxui/screen/screen.hpp>

int main() {
    using namespace ftxui;
    auto render = window(text("Only text should be colored") | color(Color::Green), text("Contents")) | color(Color::White);
    auto screen = Screen::Create(
        Dimension::Full(),
        Dimension::Fit(render)
    );
    Render(screen, render);
    screen.Print();
}
```
I have no idea how to do this correctly. I greatly appreciate and thank any help.

**Comment by ArthurSonzogni** [comment]
Thanks!

This looks like a bug. I believe this is because the "title" is flexible by default and covers the whole line, causing the `color` decorator to apply to everything that is behind.

We probably want to fix this.
